mflowgen/full_chip/glb_top/get_glb_top_outputs.sh

```diff
@@ -9,7 +9,7 @@ else
 fi
 
 mkdir -p outputs
-cp -L *cadence-genus-genlib/outputs/design.lib outputs/glb_top_tt.lib
```

Comment:
Notice the mismatch between `cadence-genus-genlib` specified here vs. `cadence-innovus-genlib` in `glb_top/construct.py` below, which apparently was also genus up until about Feb. 25. I went for a generic fix by wildcarding the entire prefix so that it works for both genus and innovus, but it's fine with me if you want to go a different way...

Reply:
huh won't this just break again when/if we switch from innovus back to genus?